README.rst

```diff
@@ -3,9 +3,101 @@ Open edX Devstack |Build Status|
 
 Tahoe Devstack Docs
 ===================
-To run a Tahoe devstack, follow the same instructions in the "Prerequisites" section,
-but use ``$ make tahoe.up`` instead of ``$ make dev.up`` to use the Tahoe specific
-features and environment settings.
+To run a Tahoe devstack follow the steps below:
+
+This project requires **Docker 17.06+ CE**.
+
+.. code::
+
```

Comment:
Are we cloning this repo? Which should be our default branch?

Reply:
The clone part is needed, thanks @johnbaldwin. But the `make dev.checkout` isn't.